src/Elgentos/Masquerade/DataProcessor/DefaultDataProcessorFactory.php

```diff
@@ -24,7 +24,8 @@ public function create(
 
         $whereCondition = $tableConfiguration['where'] ?? '';
 
-        foreach ($tableConfiguration['columns'] as $column => $columnData) {
+        $columns = $tableConfiguration['columns'] ?? [];
```

Comment:
Fixes `PHP Notice:  Undefined index: columns in src/Elgentos/Masquerade/DataProcessor/DefaultDataProcessorFactory.php on line 27` when doing a full table delete (so without columns).

```
creditmemo:
  sales_creditmemo:
    provider:
      delete: true
```